guides/installation/overview.md

```diff
@@ -81,6 +81,7 @@ SQL
 JavaScript
 
 * Node.js 12.21.0 or higher
+  * Node.js 16 is not compatible at the moment
```

Comment:
on mac :wink:

Reply:
Shall I add that OS hint?

Reply:
When it's mac os only yes. There are not so many Mac users.

Reply:
I don't know. I don't have stats about this

Reply:
Other colleagues that do not have a mac can't work with node 16.10. Didn't seem to be a mac issue for me

Reply:
Which Shopware version are you using? I guess you have still the node-sass dependency

Reply:
6.3

Reply:
Then its the wrong target branch. Master is only for 6.4 and newer

Reply:
I should rebase :D

Reply:
Might want to change this to "May not be compatible with..." since not everyone seems to have issues with Node.js 16